feature-group-definitions/custom-elements.yml

```diff
@@ -0,0 +1,14 @@
+# Tests don't allow the more specific https://html.spec.whatwg.org/multipage/custom-elements.html (yet)
+spec: https://html.spec.whatwg.org/multipage/
+caniuse: custom-elementsv1
+compat_features:
+  - css.at-rules.containe
+  - css.properties.container-nam
+  - api.CustomElementRegistr
+  - api.CustomElementRegistry.builtin_element_suppor
+  - api.CustomElementRegistry.defin
+  - api.Window.customElement
+  - css.selectors.define
+  - css.selectors.hos
+  - css.selectors.host-contex
+  - css.selectors.par
```

Comment:
These also seem to be cut off.

Reply:
Haha, now that I look at the file content in more details, every line seems to have a missing character.

Reply:
Oh wow, I have no idea how that happened. I also managed to copy and paste some container queries features from spreadsheet too. 🤦  I'll fix this shortly (then come back to your other insightful comments, after giving Chris a little more time to chime in).